apps/docs/src/pages/docs/guidelines.mdx

```diff
@@ -1,50 +1,47 @@
 # Component Guidelines
 
-This guide aims to provide insights and recommendations for building and contributing
-robust and adaptable components that are consistent with the existing UI Kit library.
+This section outlines the standards for building consistent, accessible, and maintainable components within the **UI Kit** — covering structure, styling, testing, and documentation best practices.
 
-## File Structure & Purpose
+## Structure
 
-The directory should contain the following files:
+Each component should follow the structure below:
 
-```sh
+```bash
 HvComp/
-├── index.ts            # entry file with the component's public exports
-├── HvComp.tsx          # component implementation and types
-├── HvComp.stories.tsx  # component documentation and examples
-├── HvComp.styles.ts    # component classes and styling utils
-└── HvComp.test.tsx     # component unit tests
+├── index.ts            # Entry file with public exports
+├── HvComp.tsx          # Component implementation and types
+├── HvComp.stories.tsx  # Storybook stories and documentation
+├── HvComp.styles.ts    # Styling utilities and class definitions
+└── HvComp.test.tsx     # Unit tests
 ```
 
-## Guidelines
+- Components and their types **must be exported** in `index.ts`.
+- Use **PascalCase** for component names and **prefix with `Hv`** (e.g., `HvButton`, `HvAccordion`).
```

Comment:
not a big deal, but now there's a mix of active & passive voice in a few places (_ido this_ vs _ this should be done_)